ffwd-client/src/main/java/com/spotify/ffwd/v1/Metric.java

```diff
@@ -149,10 +132,11 @@ public Metric attributes(Map<String, String> attributes) {
         builder.setValue(Protocol1.Value.newBuilder().setDoubleValue(doubleValue.getValue()));
       } else if (value instanceof Value.DistributionValue) {
         Value.DistributionValue distributionValue = (Value.DistributionValue) value;
-        ByteString byteString = ByteString.copyFrom(distributionValue.getValue());
+        ByteString byteString = ByteString.copyFrom(distributionValue.getValue().array());
```

Comment:
Is array() method in this case just return an array of bytes?

Reply:
Yes, it does